outputs.tf

```diff
@@ -1,11 +1,3 @@
-output "id" {
```

Comment:
Maybe we can remove them in the separated commit, before changes to other files?